src/common/framework/myCFG.ml

```diff
@@ -42,19 +42,34 @@ sig
   include CfgForward
 end
 
+(** Type of CFG "edges": keyed by 'from' and 'to' nodes,
+    along with the list of connecting instructions. *)
+module CfgEdge = struct
+  type t = Node.t * edges * Node.t [@@deriving eq, hash]
+end
+
+module CfgEdgeH = BatHashtbl.Make (CfgEdge)
+
+module type CfgBidirSkip =
+sig
+  include CfgBidir
+  val skippedByEdge: node -> edges -> node -> stmt list
```

Comment:
We should probably comment what this is good for here.

Reply:
And probably also make clear that this is in the forward direction, as the module here is a `CfgBidir`.